src/cms/templates/imprints/imprint_form.html

```diff
@@ -0,0 +1,475 @@
+    <script src="{% static 'tinymce/plugins/lists/plugin.js' %}"></script>
+    <script src="{% static 'tinymce/plugins/directionality/plugin.js' %}"></script>
+    <script src="{% static 'tinymce-i18n/langs/de.js' %}"></script>
+    <script src="{% static 'js/copy-clipboard.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+<form enctype="multipart/form-data" method="post" id="content_form">
+    {% csrf_token %}
+    <div class="flex flex-wrap">
+        <div class="w-4/5 flex flex-wrap flex-col justify-center mb-6">
+            <h2 class="heading font-normal">
+                {% if imprint %}
+                    {% if imprint_translation_form.instance.id %}
+                        {% with imprint_translation_form.instance.title as page_title %}
+                        {% blocktrans %}Edit Imprint "{{ page_title }}"{% endblocktrans %}
+                        {% endwith %}
+                        {% get_current_language as LANGUAGE_CODE %}
+                        {% unify_language_code LANGUAGE_CODE as LANGUAGE_CODE %}
+                        {% if LANGUAGE_CODE != language.code %}
+                            {% get_translation page LANGUAGE_CODE as backend_translation %}
+                            {% if backend_translation %}
+                                ({% trans 'Title in' %} {% translated_language_name LANGUAGE_CODE %}: "{{ backend_translation.title }}")
+                            {% endif %}
+                        {% endif %}
+                    {% else %}
+                        {% trans 'Create new translation' %}
+                    {% endif %}
+                {% else %}
+                    {% trans 'Create new imprint' %}
+                {% endif %}
+            </h2>
+        </div>
+        <div class="w-1/5 flex justify-end mb-6">
+        {% has_perm 'cms.edit_page' request.user page as can_edit_page %}
+        {% if not page_form.instance.id or not page_form.instance.archived %}
+            {% if can_edit_page %}
+                <input type="submit" name="submit_draft" class="bg-gray-500 hover:bg-gray-600 cursor-pointer text-white font-bold py-3 px-4 rounded mr-2" value="{% trans 'Save as draft' %}" />
+            {% endif %}
+            {% has_perm 'cms.publish_page' request.user page as can_publish_page %}
+            {% if can_publish_page %}
+                <input type="submit" name="submit_public" class="cursor-pointer bg-blue-500 hover:bg-blue-600 text-white font-bold py-3 px-4 rounded mr-2" value="{% trans 'Publish' %}" />
+            {% else %}
+                <input type="submit" name="submit_review" class="cursor-pointer bg-blue-500 hover:bg-green-600 text-white font-bold py-3 px-4 rounded mr-2" value="{% trans 'Submit for review' %}" />
+            {% endif %}
+            <button type="button" onclick="short_url_to_clipboard('s/p/{{ page_translation_form.instance.id }}');" class="cursor-pointer bg-blue-500 hover:bg-blue-600 text-white font-bold py-3 px-4 rounded">
```

Comment:
Not sure if we need a short URL at all for the imprint. But if we have it, we need to change the path. I'd say `i` for imprint? -> `s/i/IMPRINT_TRANSLATION_ID`.